File: src/main/java/com/uwetrottmann/trakt5/enums/IdType.java
package com.uwetrottmann.trakt5.enums;

public enum IdType implements TraktEnum {

    TRAKT("trakt"),
    IMDB("imdb"),
    TMDB("tmdb"),
    TVDB("tvdb"),
    TVRAGE("tvrage");

    private final String value;

    IdType(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }

}

File: src/main/kotlin/com/redissi/trakt/enums/IdType.kt
package com.redissi.trakt.enums

enum class IdType(private val value: String) : TraktEnum {
    TRAKT("trakt"), IMDB("imdb"), TMDB("tmdb"), TVDB("tvdb"), TVRAGE("tvrage");

    override fun toString(): String {
        return value
    }

}